Update evaluation_logger.md (#4609)
wandb/weave: docs/docs/guides/evaluation/evaluation_logger.md

## Basic workflow

1. _Initialize the logger:_ Create an instance of `EvaluationLogger`, optionally providing metadata about the `model` and `dataset`. Defaults will be used if omitted.

>     :::important Track token usage and cost
>     To capture token usage and cost for LLM calls (e.g. OpenAI), initialize `EvaluationLogger` before any LLM invocations**. 
>     If you call your LLM first and then log predictions afterward, token and cost data are not be captured.

Reviewer: ```suggestion
    :::important Track token usage and cost
    To capture token usage and cost for calls to LLMs such asOpenAI, initialize `EvaluationLogger` **before making any any LLM calls**. 
    Otherwise, if you call your LLM first and then log predictions afterward, token and cost data are not captured.
```

>     If you call your LLM first and then log predictions afterward, token and cost data are not be captured.

Author: ```suggestion
    If you call your LLM first and then log predictions afterward, token and cost data are not captured.
```
